server/models/bill.coffee

```diff
@@ -2,17 +2,22 @@ cozydb = require 'cozydb'
 
 
 module.exports = Bill = cozydb.getModel 'Bill',
-    type: String
-    subtype: String
-    date: Date
-    vendor: String
-    amount: Number
-    plan: String
-    pdfurl: String
-    binaryId: String
-    fileId: String
-    content: String
-    isRefund: Boolean
+    type: String  # Main type of the bill (check already used types in konnectors)
+    subtype: String  # Subtype of the bill (check already defined subtypes)
+    date: Date  # Date of the billing
+    vendor: String  # Third party from which the bill originates
+    amount: Number  # Bill amount
+    vat: Number  # VAT part on the bill
+    currency: String  # Currency of the bill
+    plan: String  # TODO
+    pdfurl: String  # Link to the bill document (not necessarily a PDF file)
+    binaryId: String  # TODO
+    fileId: String  # TODO
```

Comment:
It is the id of the file doctype containing the bill. The bill is just a file + metadata. Binaryid it the id of the file content